kDrive/UI/View/Menu/MainTabBar.swift

```diff
@@ -163,6 +165,30 @@ final class MainTabBar: UITabBar {
         centerButton.addTarget(self, action: #selector(centerButtonAction), for: .touchUpInside)
     }
 
+    private func setupGestureRecognizer() {
+        let longTouch = UILongPressGestureRecognizer(target: self,
+                                                     action: #selector(Self.handleLongTouch(recognizer:)))
+        addGestureRecognizer(longTouch)
+    }
+
+    @objc func handleLongTouch(recognizer: UITapGestureRecognizer) {
+        guard recognizer.state == .began else {
+            return
+        }
+
+        // Touch is over the 5th button's x position
+        let touchPoint = recognizer.location(in: self)
+        guard touchPoint.x > bounds.width / 5 * 4 else {
+            return
+        }
+
+        let generator = UIImpactFeedbackGenerator(style: .light)
+        generator.impactOccurred()
```

Comment:
A `light` impact felt more refined than a medium or a heavy on iP8.

I also think it adds to the experience on the long touch only